demisto_sdk/commands/download/downloader.py

```diff
@@ -691,9 +691,8 @@ def update_data(file_path_to_write: str, file_path_to_read: str, file_ending: st
             file_data: dict = get_json(file_path_to_write)
             if pack_obj_data:
                 merge(file_data, preserved_data)
-            json_depth: int = get_depth(file_data)
             with open(file_path_to_write, 'w') as jf:
-                json.dump(obj=file_data, fp=jf, indent=json_depth)
+                json.dump(obj=file_data, fp=jf, indent=4)
```

Comment:
Why have we abandoned the getJson function? I remember we needed it in the past

Reply:
are you talking of `get_depth` function? if so then yes, it was a mistake to use it.